src/python/pull_gsheets_translations_mvp.py

```diff
@@ -0,0 +1,136 @@
+import json
+import gspread
+from google.oauth2.service_account import Credentials
+import pandas as pd
+import os
+import string
+
+# Connect to Google Sheets API #
+# follow https://medium.com/@CROSP/manage-google-spreadsheets-with-python-and-gspread-6530cc9f15d1
+# Create credentials and dump into referenced file
+
+# Initialize contstants
+scope = ['https://spreadsheets.google.com/feeds',
+'https://www.googleapis.com/auth/drive']
+
+CREDENTIALS_PATH = '../../.my.goauth'
+TRANSLATION_SHEETS_REGEX = ' - Master Sheet'
+TRANSLATION_SHEETS_REGEX_OLD = "(OLD)"
+OUT_DIR = '../../client/public/locales/'
+#OUT_DIR = 'translations/'
+LANGUAGES_TO_PULL = ['English','Dutch (Netherlands)','Spanish','Italian','French','Russian']
+LANGUAGE_LETTERS_DICT = {
+	'English':'en',
+	'Dutch':'nl',
+	'Spanish':'es',
+	'Italian':'it',
+	'French':'fr',
+	'Russian':'ru'
+}
+
+# Google sheets authorization
+
+credentials = Credentials.from_service_account_file(CREDENTIALS_PATH, scopes=scope)
+gc = gspread.authorize(credentials)
+
+
+# Get names of spreadsheets 
+list_all_spreadsheets = gc.list_spreadsheet_files()
+list_all_spreadsheets_name = [x['name'] for x in list_all_spreadsheets]
+
+# Get names of translations
+translation_sheet_names = [ x for x in list_all_spreadsheets_name if (TRANSLATION_SHEETS_REGEX in x) & (TRANSLATION_SHEETS_REGEX_OLD not in x)]
+translation_sheet_names_select = [ x for x in translation_sheet_names for y in LANGUAGES_TO_PULL if y in x]
+translation_sheets = {}
+
+for name in translation_sheet_names_select:
+	language = '_'.join(name.split(' ')).replace('/','_')
+	translation_sheets[language] = gc.open(name).worksheets()
+
+languages = [x for x in translation_sheets.keys()]
+languages.sort()
+
+
+"""
+json structure
+{
+	"parentKey":{
+		"childKey":{
+			'fieldKey':{'value':'translatedValue','array':['translatedValue']},
+		}
+	},
+}
+"""
+
+def convert_to_camelCase(value):
+	#converts value to camelCase
+	camelCase = value.split()[0].lower() + " ".join(value.split()[1:]).title().replace(" ","")
+	text_tokens = word_tokenize(text)
+	#removes punctuation
+	return camelCase.translate(str.maketrans('', '', string.punctuation))
+
+import nltk

```

Comment:
This is just stylistic and I'm not a Python expert so correct me if I'm wrong, but I feel it's best practice to keep all imports at the top of a file

Reply:
changed to top of script